src/main/host/process.c

```diff
@@ -101,6 +101,9 @@ struct _Process {
     SimulationTime startTime;
     SimulationTime stopTime;
 
+    /* absolute path to the process' working directory */
```

Comment:
Nit - process's. A trailing ' is only used with plural nouns. e.g. processes'

Reply:
Fixed